frontend/src/metabase/setup/utils.ts

```diff
@@ -47,10 +47,19 @@ export const validatePassword = async (password: string) => {
   }
 };
 
-export const subscribeToNewsletter = async (email: string): Promise<void> => {
+export const subscribeToNewsletter = (email: string) => {
   const body = new FormData();
   body.append("EMAIL", email);
   body.append(SUBSCRIBE_TOKEN, "");
 
-  await fetch(SUBSCRIBE_URL, { method: "POST", mode: "no-cors", body });
+  if ("sendBeacon" in navigator) {
+    navigator.sendBeacon(SUBSCRIBE_URL, body);
+  } else {
+    fetch(SUBSCRIBE_URL, {
+      method: "POST",
+      mode: "no-cors",
+      body,
+      keepalive: true,
```

Comment:
I don't think there are browser that don't support sendBeacon but do support keepalive, but I guess it won't hurt?

Reply:
1. I think this is the only case where we need it, I don't think we have any navigation that's not client side inside of the app, I would keep it simple until we have another use case.
2. That's a good question, I believe our POST helper is meant to be used only for calls to our BE, it also didn't use fetch the last time I checked